depends/packages/qt.mk

```diff
@@ -92,6 +92,8 @@ $(package)_config_opts_linux += -no-opengl
 $(package)_config_opts_arm_linux += -platform linux-g++ -xplatform bitcoin-linux-g++
 $(package)_config_opts_i686_linux  = -xplatform linux-g++-32
 $(package)_config_opts_x86_64_linux = -xplatform linux-g++-64
+$(package)_config_opts_aarch64_linux = -xplatform linux-aarch64-gnu-g++
```

Comment:
This shouldn't be needed after https://github.com/bitcoin/bitcoin/pull/13696#issuecomment-406105319.

Reply:
This is not my commit, just based on #13696